artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/amq/AMQSession.java

```diff
@@ -306,8 +306,6 @@ public int sendMessage(MessageReference reference,
                           ServerConsumer consumer,
                           int deliveryCount) {
       AMQConsumer theConsumer = (AMQConsumer) consumer.getProtocolData();
-      //clear up possible rolledback ids.
-      theConsumer.removeRolledback(reference);
```

Comment:
I'm not 100% certain why this was added in the first place, but I think it would be safer to leave it here. As far as I can tell it's not hurting anything.